linux/vcpkg_config_manager/vcpkg.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Why folder name contains 'config'?

Reply:
The same for all the folders that holding the manifest vcpkg json file. It a config for vcpkg.

Reply:
Ah, you are right, my bad. I thought we have just named them vcplg_*